.github/workflows/include_supported-versions.yml

```
@@ -0,0 +1,50 @@
+name: Load Supported Versions
+
+on:
+  workflow_call:
+    outputs:
+      matrix:
+        description: "Matrix of supported versions"
+        value: ${{ jobs.load_versions.outputs.matrix }}
+      azcli_version:
```

```suggestion
      azcli_versions:
```

Beware of references elsewhere in the code